java/runtime-native/src/main/java/org/ray/spi/impl/DefaultLocalSchedulerClient.java

```diff
@@ -221,12 +222,24 @@ public static ByteBuffer taskSpec2Info(TaskSpec task) {
     // resources required by this task. The index in this vector corresponds to
     // the resource type defined in the ResourceIndex enum. For example,
 
-    int[]requiredResourcesOffsets = new int[1];
-    for (int i = 0; i < requiredResourcesOffsets.length; i++) {
-      int keyOffset = 0;
-      keyOffset = fbb.createString(ByteBuffer.wrap("CPU".getBytes()));
-      requiredResourcesOffsets[i] = ResourcePair.createResourcePair(fbb, keyOffset, 0.0);
+    final String CPULiteral = "CPU";
```

Comment:
Is it the intent to update taskSpec as well as requiredResourcesOffsets?

Reply:
Yep